### ISSUE TITLE
Support for Azure OpenAI Service?

### ISSUE THREAD

**Issue by rcarmo** [hkjang/node-red-contrib-openai#4]
Hi there,

I wanted to know if there are any plans to include support for this (which essentially boils down to a custom endpoint URL)

**Comment by vico93** [comment]
Yeah a customizable baseURL parameter would be essential to use with other OpenAI-like APIs.
